app/needToKnow/post.controller.js

```diff
@@ -0,0 +1,45 @@
+const { body } = require('express-validator')
+const { setSentencePlanComment } = require('../../common/data/sentencePlanningApi')
+const { getNeedToKnow } = require('./get.controller')
+const { countWords } = require('../../common/utils/util')
+
+const wordsAllowed = 250
+let tooManyWords = false
+
+const validationRules = () => {
+  return [
+    body('needtoknow')
+      .isLength({ min: 1 })
+      .withMessage('Add any other things the individual needs us to know about them'),
+    body('needtoknow')
+      .custom(value => {
+        tooManyWords = countWords(value) > wordsAllowed
+        return !tooManyWords
+      })
+      .withMessage('They need us to know must be 250 words or fewer'),
+  ]
+}
```

Comment:
This function is used twice now. Worth creating a utility and just passing in the text?

Reply:
We could do that but feels like a bit too much for these cases - they are the same at the moment but I'm not convinced we should be coupling them together. If we end up with a whole stack of these that really are identical then I'd agree we should change them.